ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -345,7 +345,10 @@ object BigDiffy extends Command {
               case _ =>
               DeltaValueBigQuery("UNKNOWN", None)
             }
-            DeltaBigQuery(d.field, d.left.toString, d.right.toString, dv)})
```

Comment:
I think this is probably fine, but it seems like a lot of code for such a small fix here. Technically our handling of Options is "more" correct, but we are also now modifying the underlying value and taking on responsibility to handle it correctly.

https://github.com/spotify/ratatool/pull/154/files#diff-2a46b923154e83ede309229ad1d38357R326 still passes right?

Reply:
Almost all of the code change is wrapping expected values in tests, if that makes you feel better! And, in my opinion, it was always our responsibility to handle the underlying values correctly. If it wasn't, I don't think this NPE would be a bug to fix :)

https://github.com/anne-decusatis/ratatool/blob/fix_bigdiffy_npe/ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/Diffy.scala#L68 via https://github.com/anne-decusatis/ratatool/blob/fix_bigdiffy_npe/ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala#L76 should be where that line above is used. so now the values will be wrapped in `Some(...)` but it should still work, I think. I can unwrap them and replace with `""` like the others do if it's a problem, but personally I think it's clearer to not have empty strings passed around. what do you think?

Reply:
Should keep the output behaviour consistent if possible. Otherwise, people consuming output for analysis also need to be aware of this change.

Reply:
great, I will update to unwrap in the toString